client/index.js

```
@@ -31,12 +31,14 @@ if (!fs.existsSync(PROTOCOL_VERSIONS)) {
   throw Error('Missing protocol versions file: ' + PROTOCOL_VERSIONS);
 }
 const VERSION_LIST = JSON.parse(fs.readFileSync(PROTOCOL_VERSIONS));
-if (!VERSION_LIST[CURRENT_PROTOCOL_VERSION]) {
+const MAJOR_MINOR_VERSION =
+    `${semver.major(CURRENT_PROTOCOL_VERSION)}.${semver.minor(CURRENT_PROTOCOL_VERSION)}`;
+if (semver.valid(MAJOR_MINOR_VERSION) && !VERSION_LIST[MAJOR_MINOR_VERSION]) {
```

Should we throw an error for the `!semver.valid(..)` case?

---

Separated em! Thanks for the comment!

---

Thanks! 👍